assets/blueprints/genesis_helper/src/lib.rs

```diff
@@ -75,7 +73,6 @@ mod genesis_helper {
 
             Self {
                 epoch_manager,
-                rounds_per_epoch,
```

Comment:
_(note from author)_

This was not used, so I deleted it, and then had to regenerate assets (`.sh`) and then apply some assert adjustments to unrelated metering tests 🤷‍♂️